lib/ASRouter.jsm

```diff
@@ -47,7 +49,7 @@ const MessageLoaderUtils = {
    * @returns {Array} the array of messages
    */
   _localLoader(provider) {
-    return provider.messages;
+    return provider.messages || [];
```

Comment:
I would prefer if we didn't have the fallback, that way we can fail early in case we do something wrong. Was this change required only for tests?

Reply:
Yes, this happened because of improperly formated test data.

You're right, the better thing to do here is to throw an error for the missing `.messages` property